examples/Cassie/cassie_utils.h

```diff
@@ -51,21 +51,39 @@ void addCassieMultibody(drake::multibody::MultibodyPlant<double>* plant,
     std::string filename = "examples/Cassie/urdf/cassie_v2.urdf",
     bool add_leaf_springs = true,  bool add_loop_closure = true);
 
+double AchillesLength();
```

Comment:
I think the best way to return a constant double is to list it as a `static constexpr double` member variable. Might be useful making it public just in case we need to call it again.

Reply:
done

Reply:
Can you give it a better name? https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
Ah yes. Thanks for the reference!